src/main/java/reposense/parser/ZoneIdArgumentType.java

```diff
@@ -0,0 +1,26 @@
+package reposense.parser;
+
+import java.time.DateTimeException;
+import java.time.ZoneId;
+
+import net.sourceforge.argparse4j.inf.Argument;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+import net.sourceforge.argparse4j.inf.ArgumentType;
+
+/**
+ * Verifies and parses a string-formatted zone id to a {@code ZoneId} object.
+ */
+public class ZoneIdArgumentType implements ArgumentType<ZoneId> {
+    private static final String MESSAGE_TIMEZONE_INVALID =
+            "The timezone provided is invalid, please use the format of UTC±[hh]";
+
+    @Override
+    public ZoneId convert(ArgumentParser parser, Argument arg, String value) throws ArgumentParserException {
+        try {
+            return ZoneId.of(value);
```

Comment:
It is still possible for users to choose other timezone besides UTC (like GMT) though... Do we want to strictly enforce UTC (users don't need to type UTC at all)?

Reply:
Actually, all zoneIds that can be recognized by `ZoneId` are OK, but we'd better just instruct our user to use `UTC`. Anyway, more users will not specify this arg and will use the default system time.

Reply:
>all zoneIds that can be recognized by `ZoneId` are OK, but we'd better just instruct our user to use `UTC`.

The thing is the UserGuide, DeveloperGuide and your help message as well all seems to suggest that only UTC timezone can be used, but in actual other timezones can be used as well.

Reply:
So, do you suggest that I need to ban all the non `UTC` usage of it?

Reply:
@yamidark 
Please kindly give your suggestion so I am able to progress with this pr.

Reply:
@fzdy1914 Sorry I completely missed this. I think you could probably just update the docs/help messages to give `UTC` as an example, but allow them to choose whatever timezone they may want to give instead, as long as it is valid.